<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>750</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/images/app.ico</normaloff>:/images/images/app.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Tab_MainWindow">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::TabShape::Rounded</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1250</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Tools </string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="act_RAW_Tuning_Tool"/>
    <addaction name="separator"/>
    <addaction name="act_YUV_Viewer"/>
    <addaction name="separator"/>
    <addaction name="act_RGB_Viewer"/>
    <addaction name="separator"/>
    <addaction name="act_linkboard"/>
    <addaction name="separator"/>
    <addaction name="act_Exit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Help </string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <widget class="QMenu" name="menuContact_Us">
     <property name="title">
      <string>Contact  Us</string>
     </property>
     <addaction name="actionauthor_YDJ"/>
    </widget>
    <addaction name="act_About"/>
    <addaction name="separator"/>
    <addaction name="menuContact_Us"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme </string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="act_Dark_Theme"/>
    <addaction name="separator"/>
    <addaction name="act_Light_Theme"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuTheme"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="act_RAW_Tuning_Tool">
   <property name="text">
    <string>RAW  Tuning Tool</string>
   </property>
   <property name="toolTip">
    <string>ISP全流程调试</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="act_YUV_Viewer">
   <property name="text">
    <string>YUV  Image  Viewer</string>
   </property>
   <property name="toolTip">
    <string>YUV图像查看器</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="act_linkboard">
   <property name="text">
    <string>Connect  to  Board</string>
   </property>
   <property name="toolTip">
    <string>硬件参数配置</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="act_Exit">
   <property name="text">
    <string>Exit </string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="act_DNG2RAW">
   <property name="text">
    <string>DNG2RAW</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="act_About">
   <property name="text">
    <string>About  MCISP</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="act_RGB_Viewer">
   <property name="text">
    <string>RGB  Image  Viewer</string>
   </property>
  </action>
  <action name="act_Dark_Theme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark Theme</string>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="act_Light_Theme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Light Theme</string>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionauthor_YDJ">
   <property name="text">
    <string>author@YDJ</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
